.gitlab-ci.yml

```
@@ -607,6 +610,12 @@ workflow:
     allow_failure: true
   - when: on_success
 
+# Alternative of the above to have an agent-release-management trigger
+# for stable branches in case of failure in any previous job
+.on_deploy_stable_on_failure:
+  - <<: *if_deploy_stable
+  - when: on_failure
```

```suggestion
  - <<: *if_deploy_stable
    when: on_failure
```
Isn't it what you want? Otherwise the job will be triggered on every deploy pipeline AND on every pipeline where there is a failure in its needs

---

Damn syntax, you are right, thanks a lot